src/operations/query/convertGraphQLParameters.js

```diff
@@ -41,25 +58,52 @@ function convertGraphQLParameters(queryParameterValue, args, queryParameter) {
                                 tokenString += token['value'];
                             }
                             if (tokenString) {
-                                queryParameterValue.push(tokenString);
+                                if (useNotEquals || innerNotEquals) {
+                                    notQueryParameterValue.push(tokenString);
+                                } else {
+                                    newQueryParameterValue.push(tokenString);
+                                }
                             }
                         }
                     }
+                    queryParameterValue = newQueryParameterValue;
                     break;
                 case 'reference':
                     // eslint-disable-next-line no-case-declarations
                     let referenceText = '';
+                    if (queryParameterValue['notEquals']) {
+                        const notEqualsObject = queryParameterValue['notEquals'];
+                        if (notEqualsObject['value']) {
+                            queryParameterValue['value'] = notEqualsObject['value'];
+                        }
+                        if (notEqualsObject['target']) {
+                            queryParameterValue['target'] = notEqualsObject['target'];
+                        }
+                        useNotEquals = true;
+                    }
                     if (queryParameterValue['target']) {
                         referenceText = queryParameterValue['target'] + '/';
                     }
                     if (queryParameterValue['value']) {
                         referenceText += queryParameterValue['value'];
                     }
-                    queryParameterValue = referenceText;
+                    if (useNotEquals) {
+                        notQueryParameterValue = referenceText;
+                        queryParameterValue = [];
+                    } else {
+                        queryParameterValue = referenceText;
+                    }
                     break;
                 case 'quantity':
                     // eslint-disable-next-line no-case-declarations
                     let quantityString = '';
+                    if (queryParameterValue['notEquals']) {
+                        const notEqualsObject = queryParameterValue['notEquals'];
+                        Object.keys(notEqualsObject).forEach(key => {
+                            queryParameterValue[key] = notEqualsObject[key];
```

Comment:
Please remove this warning

Reply:
Disabled, since the graphql schema limits what keys can be present, it doesn't present a security risk.